models/stdp_triplet_synapse.h

```diff
@@ -44,11 +44,12 @@ stdp_triplet_synapse is a connection with spike time dependent
 plasticity accounting for spike triplet effects (as defined in [1]_).
 
 Notes:
+
 - Presynaptic traces r_1 and r_2 of [1]_ are stored in the connection as
   Kplus and Kplus_triplet and decay with time-constants tau_plus and
   tau_plus_triplet, respectively.
 - Postsynaptic traces o_1 and o_2 of [1]_ are acquired from the postsynaptic
-  neuron states Kminus_ and triplet_Kminus_ which decay on time-constants
+  neuron states Kminus\_ and triplet_Kminusi_ which decay on time-constants
```

Comment:
Also in the lines around this one
```suggestion
  neuron states ``Kminus_`` and ``triplet_Kminusi_`` which decay on time-constants
```